app/platform/__init__.py

```diff
@@ -4,13 +4,8 @@
 from werkzeug import Response
 from wtforms.fields.core import Field
 
+from app.common.data.interfaces import grants as data
```

Comment:
:sweating-towel-guy: not like this

Reply:
But I don't have a much better option 😭 

I think I was imagining it being more like `import app.common.data.interfaces as data` - but that means aggregating all the stuff in that module.

Otherwise this feels like maybe it should be `from app.common.data.interfaces import grants as grants_data`. Because we'll have a problem as soon as we need to import any interface module beyond the first with this setup.

But this all feels grim.

Reply:
So - yolo for now? 🫠